.github/configuration/upgrade-test-configuration.yml

```diff
@@ -110,6 +110,18 @@ upgrade-version: [{
     }
   ],
 },
+{
+  upgrade-path: [
+    {
+      version: '14.12',
+      upgrade-type: null
+    },
+    { 
+      version: target.latest,
+      upgrade-type: major
+    }
+  ],
+},
```

Comment:
Better to modify the above one from 14.11 to 14.12 instead of creating a new workflow.

Reply:
Done